doc/example/amici.ipynb

```diff
@@ -806,34 +930,9 @@
    "cell_type": "markdown",
    "metadata": {
     "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
-   },
-   "source": [
-    "### Startpoint method\n",
-    "\n",
-    "The startpoint method describes how you want to choose your startpoints, in case you do a multistart optimization. The default here is `uniform` meaning that each startpoint is a uniform sample from the allowed parameter space. The other two notable options are either `latin_hypercube` or a self defined function."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "startpoint_method = pypesto.startpoint.uniform"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "collapsed": false,
```

Comment:
I think info should be given somewhere at least. However, I'm not sure where it's best to put it. 
This notebook already gives more-detailed information about optimization, history, etc... so it doesn't feel out of place in here.
I would not put it in getting-started as I don't see it as essential enough.

Reply:
Did we deprecate this way of specifying the standpoint method? If so, I would update this then to what should be used.

Reply:
the startpoint method is now defined as a property of the problem/objective, it is not used as an argument to optimization anymore

Reply:
Could we include this change here? I don't think we show it anywhere else.